contrib/to-a11y-string/test/to-a11y-string-spec.js

```diff
@@ -0,0 +1,368 @@
+        test("nested fractions", () => {
+            const result = renderString("\\frac{1}{1+\\frac{1}{x}}");
+            // TODO: this result is ambiguous, we need to fix this
+            expect(result).toMatchInlineSnapshot(
+                `"start fraction, 1, divided by, 1, plus, start fraction, 1, divided by, x, end fraction, end fraction"`,
+            );
+        });
+
+        test("binomials", () => {
+            const result = renderString("\\binom{n}{k}");
+            expect(result).toMatchInlineSnapshot(
+                `"start binomial, left parenthesis, n, over, k, right parenthesis, end binomial"`,
+            );
+        });
+    });
+
+    describe("horizBrace", () => {
+        test("\\overbrace", () => {
+            const result = renderString("\\overbrace{1+2}");
+            expect(result).toMatchInlineSnapshot(
+                `"start overbrace, 1, plus, 2, end overbrace"`,
+            );
+        });
+
+        test("\\underbrace", () => {
+            const result = renderString("\\underbrace{1+2}");
+            expect(result).toMatchInlineSnapshot(
+                `"start underbrace, 1, plus, 2, end underbrace"`,
+            );
+        });
+    });
+
+    describe("infix", () => {
+        test("\\over", () => {
+            const result = renderString("a \\over b");
+            expect(result).toMatchInlineSnapshot(
+                `"start fraction, a, divided by, b, end fraction"`,
+            );
+        });
+
+        test("\\choose", () => {
+            const result = renderString("a \\choose b");
+            expect(result).toMatchInlineSnapshot(
+                `"start binomial, left parenthesis, a, over, b, right parenthesis, end binomial"`,
+            );
+        });
+
+        test("\\above", () => {
+            const result = renderString("a \\above{2pt} b");
+            expect(result).toMatchInlineSnapshot(
+                `"start fraction, a, divided by, b, end fraction"`,
+            );
+        });
+    });
+
+    describe("mod", () => {
+        test("\\mod", () => {
+            const result = renderString("\\mod{23}");
+            // TODO: drop the "space"
+            // TODO: collate m, o, d... we should fix this inside of KaTeX since
+            // this affects the HTML and MathML output as well
+            expect(result).toMatchInlineSnapshot(`"space, m, o, d, 23"`);
+        });
+    });
+
+    describe("overline", () => {
+        test("\\overline", () => {
+            const result = renderString("\\overline{1+2}");
+            expect(result).toMatchInlineSnapshot(
+                `"start overline, 1, plus, 2, end overline"`,
+            );
+        });
+    });
+
+    describe("relations", () => {
+        test("1 \\neq 2", () => {
+            const result = renderString("1 \\neq 2");
+            expect(result).toMatchInlineSnapshot(`"1, does not equal, 2"`);
+        });
+
+        test("1 \\ne 2", () => {
+            const result = renderString("1 \\neq 2");
```

Comment:
Should be `"1 \\ne 2"`?

Reply:
`\ne` is an alias for `\neq`.

Reply:
You have a test labeled `\neq` and another test labeled `\ne`, but both tests actually test `\neq`.  You could probably just remove the `\ne` test, or the tests should be made distinct.

Reply:
I see... I'll fix that.